aggregator/src/main/java/com/vmware/flowgate/aggregator/scheduler/job/AggregatorService.java

```diff
@@ -423,4 +539,66 @@ private void syncHostTemperatureAndHumidySensor(boolean fullSync) {
       }
    }
 
+   public Map<String,Map<String,String>> getMetricsNameAndSensorsIdMap(Map<String,Map<String,String>> metricsNameAndSensorsMap,
+         Asset sensor){
+      String positionInfo = null;
+      int position = sensor.getCabinetUnitPosition();
+      if(position == 0) {
+         positionInfo = FlowgateConstant.DEFAULT_CABINET_UNIT_POSITION;
+      }
+      positionInfo = String.valueOf(position);
```

Comment:
Please make sure when the adpater translate/save the sensor mounting side information correctly in the adapter code.